src/main/java/uk/gov/hmcts/cmc/claimstore/services/ccd/CoreCaseDataService.java

```diff
@@ -48,4 +48,12 @@ public CaseDetails save(String authorisation, Claim claim) {
                 .format("Failed storing claim in CCD store for claim %s", claim.getReferenceNumber()), exception);
         }
     }
+
+    public CaseDetails findByClaimReferenceNumber(String claimReferenceNumber, String authorisation) {
+        return null;
+    }
+
+    public CaseDetails[] findByClaimSubmitterId(String submitterId, String authorisation) {
+        return null;
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Return an empty array instead of null. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonar.reform.hmcts.net/coding_rules#rule_key=squid%3AS1168)